ambariclient/base.py

```diff
@@ -23,6 +23,7 @@
 from ambariclient import events, exceptions, utils
 
 LOG = logging.getLogger(__name__)
+LOG.addHandler(utils.NullHandler())
```

Comment:
If you reach a logging statement, but no handlers are defined (either explicitly or by `logging.basicConfig`, you will get a warning to either `stdout` or `stderr` (I can't remember which).  This commonly happens when you're running tests or doing something interactively.

The easiest way to get around this is to assign a `NullHandler` to all loggers.